### PR TITLE
Fix role declared above

### PR DESCRIPTION
**Pull request by pigletfly**
Signed-off-by: pigletfly <wangbing.adam@gmail.com>

### CODE REVIEW

File: docs/content/security.md [hunk 1]
```diff
@@ -379,7 +379,7 @@ allow {                             # Allow request if...
 identity_rights[right] {             # Right is in the identity_rights set if...
     some role
     token := tokens[input.identity]  # Token exists for identity, and...
-    role := token.roles[_]           # Token has a role, and...
+    role = token.roles[_]           # Token has a role, and...
```
Comment (srenatus) [inline docs/content/security.md:382]: Please add a whitespace so the comments line up:
```suggestion
    role = token.roles[_]            # Token has a role, and...
```

But also, could you perhaps remove the `some role` line instead? What do you think? I think it'll read more nicely.

**Review by srenatus — DISMISSED** [review]

File: docs/content/security.md (hunk 1 above)
Comment (srenatus) [reply docs/content/security.md:382]: I think the entire example needs a do-over eventually...so if you only add the space, that's fine.

Comment (anderseknert) [reply docs/content/security.md:382]: Agreed on removing `some role`. Let's not use unification in docs unless needed.

**Review by srenatus — APPROVED** [review]
Thanks!

**Pull request merged**